api/model/src/main/java/org/projectnessie/model/Operation.java

```diff
@@ -91,6 +94,15 @@ interface Put extends Operation {
     @JsonView(Views.V1.class)
     Content getExpectedContent();
 
+    /**
+     * Additional information about the operation and/or content object. If and how a Nessie server
+     * uses and handles the information depends on the server version and type of metadata (called
+     * variant).
+     */
+    @JsonInclude(Include.NON_EMPTY)
+    @JsonView(Views.V2.class)
+    List<ContentMetadata> getMetadata();
```

Comment:
Should we perhaps have a new `FetchOption` constant to control the loading of metadata in commit log (for example)?

Reply:
Yes, but so that we can have a stable v2 API, shouldn't we also prepare new `FetchOption` constants (or other means) to allow limiting the about of (unneeded) data in API responses?